python/pyspark/ml/wrapper.py

```diff
@@ -67,7 +67,10 @@ def _transfer_params_to_java(self, params, java_obj):
         paramMap = self.extractParamMap(params)
         for param in self.params:
             if param in paramMap:
-                java_obj.set(param.name, paramMap[param])
+                value = paramMap[param]
+                if isinstance(value, list):
+                    value = _jvm().PythonUtils.toSeq(value)
```

Comment:
Do we need this special treatment if we overload `w` with `JList<String>` in `StringArrayParam`?